cli/src/commands/organization/create.rs

```diff
@@ -7,9 +7,8 @@ use crate::utils::*;
 crate::clap_parser_with_shared_opts_builder!(
     #[with = addr, token]
     pub struct Args {
-        /// OrganizationID
-        #[arg(short, long)]
-        organization_id: OrganizationID,
+        /// The organization name that will be created
```

Comment:
```suggestion
        /// The organization id that will be created
```

It is expected to be a uuid ?

Reply:
The type `OrganizationID` is missleading, e.g `Scille` is a valid `OrganizationID`

https://github.com/Scille/parsec-cloud/blob/9625541d33b6d11a97d7f7fa693fceac76b3b7ab/libparsec/crates/types/src/id.rs#L289-L290